### ISSUE TITLE
Suggested option: indentation size

### ISSUE THREAD

**Issue by NattieGit** [scrumble-nl/laravel-model-ts-type#25]
First off, love the package! Nice job with it :)

Now for my suggestion, currently, the generated types use 4 spaces. It would be nice to have the option to select the number of spaces used to generate a type.

**For example:**
`php artisan types:generate --indentationSpaces=2
`
Would generate a type of:
```
type Interest = {
  id: number;
  name: string;
};
```

**Comment by Luukdewaaier** [comment]
Great to hear you love this package! We are using it extensively and it makes front end development way smoother.

We will add this feature and update the ticket once it is done.

**Comment by Luukdewaaier** [comment]
It took a bit longer than expected, it has just been merged #36 it will be in the next release.

**Comment by Luukdewaaier** [comment]
This has been implemented in the latest release 10.1.0.
